public/css/filter.css

```diff
@@ -0,0 +1,126 @@
+.filter {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: center;
+}
+
+.search-bar {
+  margin: 10px;
+}
+
+.dropdown {
+  background-color: #eee;
+  display: flex;
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+}
+
+.dropdown ul {
+  background-color: #eee;
+  border: 1px solid #005a9c;
```

Comment:
Shouldn't the border color be equal to the button background color ?

Reply:
I think that better with border color not equal  to button background. It's more beautiful ( Ithink)